app/uk/gov/hmrc/gform/dms/DmsSubmissionController.scala

```diff
@@ -31,11 +31,13 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate._
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.Destinations.DmsSubmission
 import uk.gov.hmrc.gform.submission._
 
+import scala.concurrent.ExecutionContext
+
 class DmsSubmissionController(
   fileUpload: FileUploadService,
   pdfGenerator: PdfGeneratorService,
   documentLoader: Array[Byte] => PDDocument,
-  config: AppConfig)(implicit clock: Clock)
+  config: AppConfig)(implicit clock: Clock, implicit val ex: ExecutionContext)
```

Comment:
no need for `implicit val`. First implicit marker marks all parameters in a group as implicit automatically.